addons/account/models/account_reconcile_model.py

```diff
@@ -529,18 +544,7 @@ def _get_write_off_move_lines_dict(self, residual_balance, partner_id):
             if currency.is_zero(balance):
                 continue
 
-            writeoff_line = {
-                'name': line.label,
-                'balance': balance,
-                'debit': balance > 0 and balance or 0,
-                'credit': balance < 0 and -balance or 0,
-                'account_id': line.account_id.id,
-                'currency_id': currency.id,
-                'analytic_distribution': line.analytic_distribution,
-                'reconcile_model_id': self.id,
-                'journal_id': line.journal_id.id,
-                'tax_ids': [],
-            }
+            writeoff_line = line._get_write_off_move_line_dict(balance)
```

Comment:
```suggestion
            writeoff_line = line._get_write_off_move_line_dict(balance, currency)
```
Maybe we could also pass the `currency` to avoid having to read the value for each `line` (even though it should be handled by the ORM)?